### DISCUSSION TITLE
Blog Articles Not Displaying on Homepage Layout

### DISCUSSION THREAD

**Discussion by ali13** [templaza/astroid-framework#747] [General]
Hello,

I am currently using the Astroid framework for Joomla and I am encountering an issue. When I apply a layout for the homepage, my blog articles do not display. I have tried several configurations, but the articles still do not appear.

Here is a link to a video showing the problem. A video is worth a thousand words.

[(https://www.alid.fr/blog/images/videos/video-probleme.mp4)](url)

Could you please help me resolve this issue? Any suggestions or steps to troubleshoot would be greatly appreciated.

Thank you in advance for your assistance.

Best regards,

**Comment by Chacapamac** [comment]
Your link bring a 404 (Not Found) error.
Are you sure that all your articles are the right Joomla Category and Published and the blog element call this Category?

**Comment by ali13** [comment]
Sorry I had trouble pasting the links.

try see again. [https://www.alid.fr/blog/images/videos/video-probleme.mp4](https://www.alid.fr/blog/images/videos/video-probleme.mp4).

otherwise copy and paste text. https://www.alid.fr/blog/images/videos/video-probleme.mp4

Thank you very much for your time.

**Comment by sonvnn** [comment]
Hello @ali13 ,

To create a home page and a basic blog page, you must create 2 different template_styles for them. Because the blocks of the 2 pages are basically different.

With the Home Page Preset, you must duplicate the current Template Style and then assign it to the Home Menu

With the Default Preset, you leave it as Default.

**Comment by rinka88** [comment]
You can simply disable widgets in the settings on all pages except Home
